README.md

```diff
@@ -4,47 +4,30 @@
 [![Documentation](https://docs.rs/tsunami/badge.svg)](https://docs.rs/tsunami/)
 [![Build Status](https://travis-ci.org/jonhoo/tsunami.svg?branch=master)](https://travis-ci.org/jonhoo/tsunami)
 
-`tsunami` provides an interface for running short-lived jobs and experiments on EC2 spot block
+`tsunami` provides an interface for running short-lived jobs and experiments on cloud
 instances. Most interaction with this library happens through
-[`TsunamiBuilder`](struct.TsunamiBuilder.html).
+[`TsunamiBuilder`](struct.TsunamiBuilder.html) and [`Tsunami`](struct.Tsunami.html).
 
-## Examples
+# Example
 
-```rust
-let mut b = TsunamiBuilder::default();
-b.use_term_logger();
-b.add_set(
-    "server",
-    1,
-    MachineSetup::new("m5.large", "ami-e18aa89b", |ssh| {
-        ssh.cmd("yum install nginx").map(|out| {
-            println!("{}", out);
-        })
-    }),
-);
-b.add_set(
-    "client",
-    3,
-    MachineSetup::new("m5.large", "ami-e18aa89b", |ssh| {
-        ssh.cmd("yum install wget").map(|out| {
-            println!("{}", out);
-        })
-    }),
-);
-
-b.run(|vms: HashMap<String, Vec<Machine>>| {
-    println!("==> {}", vms["server"][0].private_ip);
-    for c in &vms["client"] {
-        println!(" -> {}", c.private_ip);
-    }
-    // ...
-    Ok(())
-}).unwrap();
+```rust,no-run
+use tsunami::providers::{aws, Launcher};
+use tsunami::TsunamiBuilder;
+fn main() -> Result<(), failure::Error> {
```

Comment:
I'd like this example to have some more comments explaining what's going on, more than one machine, include a setup function, etc. This is going to be the _primary_ thing people look at to figure out how to do what they want after all

Reply:
The old main example is a good case-study:
https://github.com/jonhoo/tsunami/blob/96a2f912f445b57b02a217b4a6998837373989b6/src/lib.rs#L7-L38